robot_smach_states/src/robot_smach_states/manipulation/manipulation.py

```diff
@@ -73,117 +103,90 @@ def __init__(self, robot, arm_designator, grabbed_entity_label="", grabbed_entit
+        :param arm_designator: designator that resolves to arm holding the object
+        :param timeout: float amount of time the procedure may take
+        """
+        smach.StateMachine.__init__(self, outcomes=['succeeded', 'failed'])
 
-        #A designator can resolve to a different item every time its resolved. We don't want that here, so lock
+        # A designator can resolve to a different item every time its resolved. We don't want that here, so lock
         check_type(arm_designator, PublicArm)
         locked_arm = LockingDesignator(arm_designator)
 
         with self:
-            smach.StateMachine.add("LOCK_ARM",
-                        LockDesignator(locked_arm),
-                        transitions={'locked'         :'SPINDLE_MEDIUM'})
-
-            smach.StateMachine.add("SPINDLE_MEDIUM",
-                        ResetPart(robot, robot.torso),
-                        transitions={'done'         :'MOVE_HUMAN_HANDOVER_JOINT_GOAL'})
-
-            smach.StateMachine.add("MOVE_HUMAN_HANDOVER_JOINT_GOAL",
-                        ArmToJointConfig(robot, locked_arm, 'handover_to_human'),
-                        transitions={ 'succeeded'   :'SAY_OPEN_GRIPPER',
-                                      'failed'      :'SAY_OPEN_GRIPPER'})
-
-            smach.StateMachine.add("SAY_OPEN_GRIPPER",
-                        Say(robot, [ "Watch out, I will open my gripper in one second. Please take it from me."]),
-                        transitions={   'spoken'    :'OPEN_GRIPPER_HANDOVER'})
-
-            smach.StateMachine.add('OPEN_GRIPPER_HANDOVER', SetGripper(robot, locked_arm, gripperstate=GripperState.OPEN, timeout=2.0),
-                        transitions={'succeeded'    :   'SAY_CLOSE_NOW_GRIPPER',
-                                     'failed'       :   'SAY_CLOSE_NOW_GRIPPER'})
-
-            smach.StateMachine.add("SAY_CLOSE_NOW_GRIPPER",
-                        Say(robot, [ "I will close my gripper now"]),
-                        transitions={   'spoken'    :'CLOSE_GRIPPER_HANDOVER'})
-
-            # smach.StateMachine.add('OPEN_GRIPPER_ON_HANDOVER', OpenGripperOnHandoverToHuman(robot, locked_arm, timeout=timeout),
-            #             transitions={'succeeded'    :   'CLOSE_GRIPPER_HANDOVER',
-            #                          'failed'       :   'SAY_I_WILL_KEEP_IT'})
+            smach.StateMachine.add("LOCK_ARM", LockDesignator(locked_arm),
+                                   transitions={'locked': 'SPINDLE_MEDIUM'})
 
-            smach.StateMachine.add('CLOSE_GRIPPER_HANDOVER', SetGripper(robot, locked_arm, gripperstate=GripperState.CLOSE, timeout=0.0),
-                        transitions={'succeeded'    :   'RESET_ARM',
-                                     'failed'       :   'RESET_ARM'})
+            smach.StateMachine.add("SPINDLE_MEDIUM", ResetPart(robot, robot.torso),
+                                   transitions={'done': 'MOVE_HUMAN_HANDOVER_JOINT_GOAL'})
```

Comment:
This state is not required for HERO. Do we wanna keep this in an attempt to keep compatability with amigo (which we have already lost long ago)?

Reply:
Compatibility is always better than no compatibility and since this has no disadvantages I would say keep it in